vignettes/epidist.Rmd

```diff
@@ -1,201 +1,250 @@
 ---
-title: "Getting Started with Epidist"
-description: "A quick start guide to using the Epidist package"
-author: Epidist Team
+title: "Getting started with epidist"
+description: "A quick start guide to using the epidist R package"
 output: 
-  bookdown::html_vignette2:
+  bookdown::html_document2:
```

Comment:
Can you explain?

Reply:
I find that the `html_vignette2` format creates figures which are formatted strangely (low resolution, incorrect scale of different objects in the figure). I guess this is somewhat of a hack and there is a reason that we would want the vignette to be in `html_vignette2` format rather than `html_document2` format. If there is some way to make the figures less bad in `html_vignette2` please let me know.

Reply:
Okay fair enough. I don't have a strong preference here. I'm not clear if there is a reason for `html_vignette` apart from backwards compatible formatting and if its just that no issue not using